src/explore-education-statistics-frontend/src/components/CookieBanner.module.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+  background: #d5e8f3;
+  padding: 10px 0;
+  width: 100%;
+
+  p {
+    font-family: 'nta', Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 1.14286;
+    margin: 0 15px;
+    max-width: 960px;
+    text-transform: none;
+
+    @media (min-width: 40.0625em) {
```

Comment:
Use the Design System media query mixin here. You can easily access to these by using the helper import `~explore-education-statistics-common/styles/govuk-base`.